src/send/saga.ts

```diff
@@ -441,8 +431,6 @@ export function* sendPaymentOrInviteSaga({
           amount: amount.toString(),
         })
       }
-    } else if (recipientHasNumber(recipient)) {
-      yield call(sendInvite, recipient.e164PhoneNumber, amount, usdAmount, tokenAddress, feeInfo)
     } else {
       throw new Error('')
```

Comment:
Interesting we didn't provide a message here 🤔 
We should probably add one.

Reply:
added!